app/queries/get_host_content.rb

```diff
@@ -16,6 +16,13 @@ class GetHostContent
       :instances,
     )
 
+    Rollup = Data.define(
+      :views,
+      :locations,
+      :content_blocks,
```

Comment:
should we keep it as `instances` if that's the term on the frontend as well?

Reply:
Done! :